README.md

```diff
@@ -179,8 +179,10 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [Codacy](https://www.codacy.com) [:rocket:](https://www.codacy.com/careers) [:octocat:](https://github.com/codacy) | Automated code reviews & code analytics. | `Lisboa` `Remote` |
+| [Dash0](https://www.dash0.com/) [:rocket:](https://careers.dash0.com/) [:octocat:](https://github.com/dash0hq) | Collaborative API observability platform for debugging and monitoring APIs. | `Remote` |
```

Comment:
```suggestion
| [Dash0](https://www.dash0.com/) [:rocket:](https://careers.dash0.com/) [:octocat:](https://github.com/dash0hq) | Observability platform. | `Remote` |
```

70 char max
https://github.com/marmelo/tech-companies-in-portugal/blob/master/CONTRIBUTING.md